### DISCUSSION TITLE
plugins/

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#300] [Announcements]
# plugins/

主题在近1年多的时间里集成了非常的多功能了，接下来计划将这些能力拆分成组件，以便于更方便的与其它主题集成。

https://theme.sugarat.top/plugins/

**Comment by dakeAa123456** [comment]
请教下博主左下角的看板娘插件是怎么配置的，有相关教程吗？如有看到，望回复，感谢

**Reply by ATQQ** [reply]
https://theme.sugarat.top/config/component.html#oml2d-%E7%9C%8B%E6%9D%BF%E5%A8%98%E9%9B%86%E6%88%90

使用的主题的话可以看这里。

库是这个 https://oml2d.com/

模型资源可能得自己托管一下，淘宝源CDN现在白嫖不了

**Reply by dakeAa123456** [reply]
https://oml2d.com/  域名过期了，不能访问了，没有配置指南了 @ATQQ

**Reply by ATQQ** [reply]
可以看GitHub  https://github.com/oh-my-live2d/oh-my-live2d

**Comment by SlovinG** [comment]
一年前俺写的打赏插件，不知道能否在这贴一贴～：
https://www.npmjs.com/package/vitepress-plugin-sponsor

**Reply by ATQQ** [reply]
没问题 兄弟你自己也可以提pr合到文档里

**Reply by ATQQ** [reply]
好了，放在推荐页 https://theme.sugarat.top/recommend.html#sponsor-%E6%89%93%E8%B5%8F%E6%A8%A1%E5%9D%97

**Reply by yzg011** [reply]
可以的 我这边就用到了这个插件。 https://bj.z2m.store/

**Comment by yzg011** [comment]
https://bj.z2m.store。 这个还是可以的
